doc/api/tls.md

```diff
@@ -1031,7 +1039,9 @@ console.log(tls.getCiphers()); // ['AES128-SHA', 'AES256-SHA', ...]
 ## Deprecated APIs
 
 ### Class: CryptoStream
-
+<!-- YAML
+added: v0.3.2
```

Comment:
This should, ideally, mention when it was deprecated, i.e. an additional `deprecated: vx.y.z` line